app/models/fee/base_fee.rb

```diff
@@ -116,9 +116,7 @@ def perform_validation?
     end
 
     def calculated?
-      fee_type.calculated?
-    rescue
-      true
+      fee_type&.calculated?.nil? ? true : fee_type&.calculated?
```

Comment:
Could be simplified with the OR operator
```ruby 
fee_type.calculated? || true
```
if the call on the left fails, it will return true

Reply:
tried this and unfortunately this does not handle the `false` possibility. 
```
false || true => true
```
The code is, nonetheless, not a very nice way of defaulting nil to true.